agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -578,22 +596,35 @@ private static boolean isRuntimeAttached() {
   }
 
   public static class SamplingOverride {
-    // TODO (trask) consider making this required when moving out of preview
-    @Nullable public SpanKind spanKind;
+    @Deprecated @Nullable public SpanKind spanKind;
+
+    // TODO (trask) make this required when moving out of preview
+    //   for now the default is both "request" and "dependency" for backwards compatibility
+    @Nullable public TelemetryKind telemetryKind;
+
+    // TODO (trask) should this be named "standalone" (and meaning flipped)
+    //  especially if we aren't going to change meaning of "request" to include unparented INTERNAL
+    //  spans
+    public boolean inRequest = true;
```

Comment:
TODO

Reply:
should we rename it to isSpanValid? Essentially that is it, right?

Reply:
i don't understand why this is made configurable?

Reply:
renamed to `includeStandaloneTelemetry`